gerrit_to_github_issues/github_issues.py

```diff
@@ -61,14 +61,14 @@ def remove_duplicated_issue_numbers(issue_dict: dict) -> dict:
     return issue_dict
 
 
-def get_repo(repo_name: str, github_user: str, github_pw: str, github_token: str) -> (github.Github, Repository):
```

Comment:
Can you remove line 18? This was the last place Repository was used and is now an extraneous import